src/R/Editor/Test/Signatures/ComputeCurrentParameterTest.cs

```diff
@@ -117,15 +121,15 @@ public class ComputeCurrentParameterTest : FunctionIndexBasedTest {
         public async Task ParameterTest_ComputeCurrentParameter03() {
             await PackageIndexUtility.GetFunctionInfoAsync(FunctionIndex, "legend");
 
-            REditorSettings.PartialArgumentNameMatch = false;
+            //REditorSettings.PartialArgumentNameMatch = false;
```

Comment:
Will remove comment. false is default and now that it is not static no need to set it.